environments/ithc/ithc.tfvars

```diff
@@ -1,5 +1,5 @@
 project                    = "hmcts"
-location                   = "uksouth"
+location                   = "ukwest"
```

Comment:
I don't think you want to change this? that'll probably not work, it's not setup for multi region